src/recipe.rs

```diff
@@ -344,7 +344,10 @@ impl<'src> Display for Recipe<'src> {
         }
         match fragment {
           Fragment::Text { token } => write!(f, "{}", token.lexeme())?,
-          Fragment::Interpolation { expression, .. } => write!(f, "{{{{{}}}}}", expression)?,
+          Fragment::Interpolation { expression, .. } => match expression {
```

Comment:
I'd actually be fine with adding space around all expressions. It would remove this special case, and might be clearer. What do you think?

Reply:
Totally agree. It certainly looks cleaner to me.